content/automate/ManageIQ/System/CommonMethods/Utils.class/__methods__/dump_root.rb

```diff
@@ -0,0 +1,10 @@
+#
+# Description: Log all attributes stored in the $evm.root hash.
+#
+
+def dump_root(handle = $evm)
```

Comment:
@billfitzgerald0120 this needs the modules and also the name of a class preferably Utils.
And the dump_root should be a class method

```
def self.dump_root(handle = $evm)
....
end
```